frontend/src/components/pages/CardDetails/Presentation.jsx

```diff
@@ -200,7 +197,21 @@ export default ({
         ) : (
           <React.Fragment />
         )}
+
+        {/* 
+      We make the view content button default in this view. This slightly changes the order of the
+      buttons that are to be rendered in this page("view content button" will always be at the bottom of the other buttons)
+      The issue of this button having duplicates is resolved.
```

Comment:
Will this comment make sense once the PR is merged? Was it meant to be left in the code? 

If you  want to leave a comment for your reviewers then you could use github's comment feature.